tests/pipelines/main.nf.test

```diff
@@ -0,0 +1,28 @@
+nextflow_pipeline {
+
+    name "Integration Tests from assemblies for MIKROKONDO"
+    script "main.nf"
+
+    test("Small-scale test of full pipeline") {
+        tag "pipeline"
+
+        when {
+            params {
+                input = "$baseDir/tests/data/samplesheets/samplesheet1.csv"
+                outdir = "results"
+            }
+        }
+
+        then {
+            assert workflow.success
+            assert path("$launchDir/results").exists()
+            assert path("$launchDir/results/locidex/merged/profile.tsv").exists()
```

Comment:
My only comment is consider potentially adding something that checks the contents of `profile.tsv`.

Reply:
Good idea. Added contents check in a03c22b0176c3386198953cc66e66381ea922251